docs/source/examples/supplychain/overview.rst

```diff
@@ -0,0 +1,100 @@
+*******************
+SupplyChain Example
+*******************
+
+In this example provided the infrastructure necessary track physical objects
```

Comment:
Suggest rewording to "This example tracks physical objects on Sawtooth using the Supplychain Transaction Family."